### PR TITLE
Adding README for multiple database support

### PR DESCRIPTION
**Pull request by DukeDavis12**
Adding README for multiple database support.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

**Review by KiranSukhavasi — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Review by v-vasavi — COMMENTED** [review]
Are we calling out the database specific steps to be followed, anywhere?

**Comment by DukeDavis12** [comment]
> > Are we calling out the database specific steps to be followed, anywhere?
> 
> Hibernate-properties for 4(mariaDB, H2, postgres & mySQL) databases are mentioned here.

The only thing that varies is Hibernate-properties; the Rest of the steps remain the same.

**Review by SSWAMIN1SSS — DISMISSED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

File: component-samples/demo/README.MD [hunk 1]
```diff
@@ -458,26 +458,80 @@ This implementation will trust Ownership vouchers with X5Chain encoding without
 
 # Support for multiple Database.
 
-FDO services supports multiple database for the storage layer. Hibernate framework is used to bridge the application layer with backend database storage layer. Out of box, the following databases are supported by the FDO service:
+FDO services supports multiple database for the storage layer. Hibernate framework is used to bridge the application layer with backend database storage layer. Out of the box, the following databases are supported by the FDO service:
```
Comment (v-vasavi) [inline component-samples/demo/README.MD:461]: FDO services support multiple databases

File: component-samples/demo/README.MD [hunk 2]
```diff
@@ -458,26 +458,80 @@ This implementation will trust Ownership vouchers with X5Chain encoding without
...
+        hibernate.connection.url: jdbc:mysql://host.docker.internal:3306/fdo?createDatabaseIfNotExist=true&useSSL=false&allowPublicKeyRetrieval=true
+        hibernate.dialect: org.hibernate.dialect.MySQLDialect
+        hibernate.dialect.storage_engine: innodb
+        hibernate.connection.requireSSL: false
+        hibernate.connection.autoReconnect: true
+      ```
+
+    c. Make sure that `useSSL` and `requireSSL` is set to `false` in `service.env` file.
+
+    d. Command to start MySQL server
```
Comment (v-vasavi) [inline component-samples/demo/README.MD:489]: Use the below command to start MySQL server

File: component-samples/demo/README.MD [hunk 3]
```diff
@@ -458,26 +458,80 @@ This implementation will trust Ownership vouchers with X5Chain encoding without
...
+        hibernate.connection.autoReconnect: true
+        hibernate.connection.requireSSL: false
+        hibernate.connection.url: jdbc:postgresql://host.docker.internal:5432/fdo?createDatabaseIfNotExist=true&useSSL=false
+        hibernate.dialect: org.hibernate.dialect.PostgreSQLDialect
+      ```
+
+
+    c. Make sure that `useSSL` and `requireSSL` is set to `false` in `service.env` file.
+
+    d. Command to start PostgreSQL server
```
Comment (v-vasavi) [inline component-samples/demo/README.MD:522]: Use the below command to start PostgreSQL server

File: component-samples/demo/README.MD (hunk 1 above)
Comment (DukeDavis12) [reply component-samples/demo/README.MD]: Updated.

File: component-samples/demo/README.MD (hunk 3 above)
Comment (DukeDavis12) [reply component-samples/demo/README.MD]: Updated.

**Review by KiranSukhavasi — APPROVED** [review]

**Review by v-vasavi — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request merged**